tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es5",
     "module": "commonjs",
-    "jsx": "preserve",
+    "jsx": "react",
```

Comment:
Significant change 2

Reply:
For anyone else who wanted to see the docs on this: https://www.typescriptlang.org/docs/handbook/jsx.html#basic-usage